src/common/error.rs

```diff
@@ -187,3 +187,9 @@ impl From<std::io::Error> for Error {
         Self::Other(err.to_string())
     }
 }
+
+impl From<typeql_lang::common::Error> for Error {
+    fn from(err: typeql_lang::common::Error) -> Self {
+        Self::Other(err.to_string())
+    }
+}
```

Comment:
Again, instead of `Other` this should really be a `TypeQL(typeql_lang::common::Error)` variant. There's no sense in discarding information here.